src/MCPClient/lib/clientScripts/upload-archivesspace.py

```diff
@@ -217,9 +219,10 @@ def upload_to_archivesspace(files, client, xlink_show, xlink_actuate, object_typ
     parser.add_argument('--uri_prefix', metavar="uri prefix")
     parser.add_argument('--access_conditions', metavar="conditions governing access", default="")
     parser.add_argument('--use_conditions', metavar="conditions governing use", default="")
+    parser.add_argument('--inherit_notes', metavar="Inherit digital object notes from the parent component", default=False, type=lambda x: x.lower() in ('yes', 'y', 'true', '1'))
```

Comment:
I tried what you're saying without much luck because what I get is one of the choices, which are strings, but I want a boolean right?